src/util/allIcons.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Exports an array of all icon names.
+ *
+ * This file is generated, and you shouldn't manually modify it. It can be re-generated by
+ * running `yarn generate-icon-types`.
+ *
+ * @generated
+ */
+
+export const ALL_ICONS = [
```

Comment:
Icons derived from what was in `main`, non deleted or added to match `next`

Reply:
Does it match the icon set introduced in this PR? Is it going to be updated to match `next` in a different PR?

Reply:
should be auto-generated from the SVG files that exist

Reply:
> Does it match the icon set introduced in this PR? Is it going to be updated to match next in a different PR?

Yea it's auto generated from the icon set in the PR as Annie stated, and should be auto gen updated to match `next` when that comes